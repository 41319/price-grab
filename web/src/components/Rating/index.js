import * as React from "react"
import PropTypes from "prop-types"
import { Link } from "gatsby"
import { Star } from "grommet-icons"
import { Box } from "grommet"

const Rating = ({ rate, stars }) => {
  return (
    <Box direction="row">
      {[...Array(stars).keys()].map(e => (
        <Star key={e} color="#FFC95E" />
      ))}
      {rate}
    </Box>
  )
}

export default Rating
