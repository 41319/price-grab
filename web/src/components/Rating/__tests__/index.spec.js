import React from "react"
import renderer from "react-test-renderer"
import Rating from "../"
describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Rating />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
