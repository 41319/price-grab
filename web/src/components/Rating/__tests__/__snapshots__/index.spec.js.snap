// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 dnSXwv"
>
  <svg
    aria-label="Star"
    className="StyledIcon-ofa7kd-0 jiMfcF"
    viewBox="0 0 24 24"
  >
    <polygon
      fill="#FFC95E"
      fillRule="evenodd"
      points="12 16.667 5 22 8 14 2 9.5 9.5 9.5 12 2 14.5 9.5 22 9.5 16 14 19 22"
    />
  </svg>
</div>
`;
